src/main/target/REVO/target.h

```diff
@@ -67,14 +71,33 @@
 #define BEEPER                  NONE
 #endif
 
-// PC0 used as inverter select GPIO
-#define INVERTER_PIN_UART1      PC0
+// UART inverters
+#ifdef AIRBOTF4SD
+#  define INVERTER_PIN_UART6      PD2
```

Comment:
Not sure about mixing. Especially when there is only a small amount of indented directives, they will get overlooked when people grep for `#defined FOO` and get results. (I admit to being guilty of having done this.)